README.md

```diff
@@ -104,6 +105,8 @@ code, other repos are also under active development:
   Heroku in one click.
 - [Kong and Instaclustr](https://www.instaclustr.com/solutions/managed-cassandra-for-kong/): Let
   Instaclustr manage your Cassandra cluster.
+- [Nightly Releases](https://bintray.com/kong/kong-community-edition-nightly/kong-master): Nightly
+  releases are built by travis-ci and available on bintray.
```

Comment:
Instead of "...available on bintray." I'd prefer we were less-specific as just said "...available for download."